Docs/Debugging/Debugging with Eclipse and pyOCD.md

```diff
@@ -0,0 +1,67 @@
+#Debugging with Eclipse and pyOCD
+
+Through debugging, you can do things such as set breakpoints, set watchpoints, view registers, view disassembly, browse memory and examine the callstack. This article explains how to debug an exported mbed project with Eclipse and pyOCD. To debug a project, first, import it with mbed CLI, then create an Eclipse Makefile project and finally set up debugger configurations. You can find more detailed explanations of these steps below.
+
+##Initial setup
+1. Download the latest version of Eclipse with CDT.
+1. Install the GNU ARM Eclipse plugin.
+	1. Open Eclipse.
+	1. Click the **Help** menu item and select **Install New Software**.
+	1. In the **Work with** box, paste the install address and press Enter: http://sourceforge.net/projects/gnuarmeclipse/files/Eclipse/updates/. If this does not work for you, please see the [GNU ARM Eclipse solutions and workarounds](http://gnuarmeclipse.github.io/blog/2016/12/02/plugins-install-issue/).
+  1. The package **GNU ARM C/C++ Cross Development Tools** appears. Select it.
+  1. Click **Next** repeatedly and accept licence agreements.
+  1. Click **Finish**. If prompted to restart Eclipse, click **Yes**.
+1. [Install tools necessary for Makefile projects](https://docs.mbed.com/docs/mbed-os-handbook/en/5.3/dev_tools/third_party/):
+  1. [GCC ARM Embedded](https://launchpad.net/gcc-arm-embedded).
+    1. Windows only: add the GCC ARM Embedded's bin directory. For example, `C:\Program Files (x86)\GNU Tools ARM Embedded\4.9 2015q2\bin`.
+  1. Python
+    1. Windows Only: select "Add to path" when installing.
+1. Install pyOCD by running `pip install pyocd`.
+1. Install mbed CLI by running `pip install mbed-cli`.
+
+##Importing to Eclipse and building
+1. Import the project with `mbed <project>`.
+1. Open Eclipse.
+1. **File** > **New** > **Makefile Project with Existing Code**.
+  1. Paste the location of your project into **Existing Code Location**.
+  1. **Project Name** automatically populates. You can update the name here.
+  1. In **Toolchain for Indexer Settings**, select `<none>`.
```

Comment:
@iriark01 Fun fact: When you type **<\none>** without "\", the word disappears, even in comments.